spec/request/claim_documents_spec.rb

```diff
@@ -3,10 +3,7 @@
 require 'rails_helper'
 RSpec.describe 'Claim Document Attachment', type: :request do
   let(:file) do
-    fixture_file_upload(
-      "#{::Rails.root}/spec/fixtures/files/doctors-note.pdf",
-      'application/pdf'
-    )
+    fixture_file_upload('files/doctors-note.pdf')
```

Comment:
Do you need to specify 'application/pdf' type here as a second argument?

Reply:
nope, it's an optional arg.